### PR TITLE
Load fat jar but do not analyse dependencies [TG-9521]

### PR DESCRIPTION
**Pull request by owen-mc-diffblue**
Branch used for analysis for this benchmark is add-jacoco, see https://diffblue.atlassian.net/wiki/spaces/DOC/pages/51282069/Dashboard+benchmarks

With all dependencies on the classpath but "context-excluded", we expect the coverage to be the same as before after this change, and the analysis time should not be much longer.

We should probably wait until at least the day after https://diffblue.atlassian.net/browse/TG-9673 is done to merge this.

### CODE REVIEW

File: diffblue-base.yml [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-#
-# Other general options such as skipVerification, ignoreExistingCoverage, CBMC
-# arguments (except for java-version) and phases SHOULD NOT be specified here.
-# They are part of diffblue.yml file that is automatically generated by
-# merging a custom yml file on top of this one using the script
-# in dashboard/scripts/update-diffblue-yml. Any options specified here would
-# be overriden to keep consistency with other benchmarks.
-# mvn -X adds debugging info to the build output which is necessary to build
-# the classpath.
-buildCmd: cd src/supernode; mvn -e -X -Dmaven.test.skip=true compile
```
Comment (majakusber) [inline diffblue-base.yml:13]: How is this now taken care of now? It's true this file is no longer used but this command does not appear in the `diffblue-base.json`, please make sure this has been taken care of. @JohnLBergqvist Can you perhaps clarify this?

Comment (JohnLBergqvist) [reply diffblue-base.yml:13]: the location of the source code for the module (`src/supernode`) is specified in firebase: https://console.firebase.google.com/project/diffblue-dashboard/database/diffblue-dashboard/data/benchmarks/Dragonfly/modules

**Review by JohnLBergqvist — APPROVED** [review]

File: diffblue-base.yml (hunk 1 above)
Comment (owen-mc-diffblue) [reply diffblue-base.yml:13]: I asked him before. We just upload the jars now, so we don't need to run maven.

**Review by majakusber — APPROVED** [review]

**Comment by owen-mc-diffblue** [comment]
This should not be merged until we have a solution to the strange layout of the fat jar for Spring projects

**Pull request closed without merging**